File: Python solutions/215. Kth Largest Element in an Array.py
from queue import PriorityQueue

class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        q = PriorityQueue()

        n = len(nums)
        for num in nums:
            q.put(-num)
        
        ans = 0
        while k:
            ans = q.get()
            k -= 1
    
        return -ans
        

File: C++ solutions/215. Kth Largest Element in an Array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int> q;

        for(int i=0;i<nums.size();i++){
            q.push(nums[i]);
        }
        
        int ans = 0;
        while(k){
            ans = q.top();
            q.pop();
            k -= 1;
        }

        return ans;

    }
};